### ISSUE TITLE
동작 문의

### ISSUE THREAD

**Issue by socialskyo** [cydemo/rx-addon-ap_email_when_join#1]
1. 가입 인증을 필요로 하는 사이트에서는 유저가 받는 환영 메일 전송 기능이 동작을 안하는 듯 합니다. 
2. 가입 인증전인데 회원 가입 안내 메일을 받긴 합니다. 가능 하다면 회원 가입 인증 후에 발송이 되는게 더 맞는게 아닌가 싶기도 합니다.
(피드백 입니다.)

**Comment by cydemo** [comment]
회원에게 도착하는 가입 환영 메일 말씀이죠?
이게 (쓸데없이ㅜ) 가입 인증 전에 전송이 되고, 오히려 가입 인증 뒤에는 전송되지 않는다는 말씀이신 거죠?

만약 그렇다면 코어에서 제공하는 트리거가 제한적이어서 지금 당장 취할 수 있는 조치는 없는 것 같습니다.
말씀하신 상황이라면 매우 중요한 이슈임이 분명한데, 코어 개발진에 '가입 인증' 트리거 제공을 문의하면서 접근해야 할 것 같습니다.(어쩌면 있는데 제가 잘 못찾는 것일 수도 있겠어요)
암튼 지금 생각으로는 그렇게 코어가 업데이트가 되어야 애드온도 손을 볼 수 있지 않을까 싶은데요, 근데 그렇게 되면 코어를 2.* 버전대로 업데이트를 하셔야 하겠네요!

**Comment by socialskyo** [comment]
@cydemo 

회원에게 도착 하는 환영 이메일은 아예 안오고
관리자가 받는 알림 이메일은 가입 인증 전에 옵니다.  ^^ 

그리고 괜찮습니다. 제보 차원 이었습니다.

**Comment by cydemo** [comment]
@socialskyo 아, 회원에게 메일이 중복 전송(환영 이메일, 인증 안내 메일)되는 건 아니라니 그나마 다행이네요.
기회 있을 때 좀 더 신중하게 알아보도록 하겠습니다~

**Comment by cydemo** [comment]
@socialskyo 
메일 인증 사용의 경우 인증 성공 시에 동작하도록 해봤는데, 제대로 테스트를 못해봤어요;;;
시간 되실 때, 링크의 소스로 파일 내용을 바꿔서 테스트 부탁드립니다!
https://gist.github.com/cydemo/bf2e371c91ad181992e97bdb06303129

**Comment by socialskyo** [comment]
@cydemo 예. 가입 인증후 관리자에게 알림 메일이 오도록 변했습니다. 아직까지 가입자에게 알림 메일은 오지 않는 듯 합니다.
